R/functions-MSnExp.R

```diff
@@ -351,3 +351,28 @@ removeReporters_MSnExp <- function(object, reporters = NULL,
     return(object)
 }
 
+.chromatogram <- function(x, rt, mz, aggregationFun = "sum",
+                          missingValue = NA_real_) {
+    missingValue <- as.numeric(missingValue)
```

Comment:
Sorry, I first thought of implementing this function for `MSnExp` objects but then decided to go for one function that can be used for both `MSnExp` and `OnDiskMSnExp` (i.e. the `extractMultipleChromatograms` function. Just forgot to remove this function.